.nvmrc

```diff
@@ -0,0 +1 @@
+18.10.0
```

Comment:
Appreciate this change @aktary - I'll need to update the test suite to drop support 🤣